docs/configuration.rst

```diff
@@ -349,45 +370,80 @@ Furthermore there are two different types of world cropping:
        :align: center
        :alt: A circular cropped world.
 
+.. note::
+
+    It's best to read positions from a ``render_view = topdown``, from 
+    `Mapcrafter URLs`_, or using the ``F3`` debug screen within Minecraft, 
+    as the x and z co-ordinates in bottom left of isometric maps can be 
+    misleading. 
+
 .. note::
 
     The renderer automatically centers circular cropped worlds and rectangular
     cropped worlds which have all four limits specified so the maximum
     zoom level of the rendered map does not unnecessarily become as high as 
     the original map. 
-    
+
+.. note::
+
     Changing the center of an already rendered map is complicated and 
     therefore not supported by the renderer. Due to that you should 
     completely re-render the map when you want to change the boundaries of 
     a cropped world. This also means that you should delete the already 
     rendered map (delete ``<output_dir>/<map_name>``).
 
 
-**Other Cropping Options**
+**Block Mask Cropping**
 
-The options for world cropping are very versatile as you can see
-with the next two options:
+Block mask is an extremely powerful cropping tool to hide or show specific block
+types. It's use requires a little knowledge about how Minecraft stores block information,
+and how Mapcrafter works with blocks.
 
-**Crop Unpopulated Chunks:** ``crop_unpopulated_chunks = true|false``
+Minecraft stores two pieces of information about each block: a *block id* and optional
```

Comment:
Ha! It took me a while - usually I can break into a debugger, but I haven't tried to get a debugger working when building code in "Linux" while editing in Windows. Having said that, tracing through foreign code to fix obscure bugs pretty common in my professional life!

Reply:
Using `minecraft:oak_wood` instead of arbitrary integers is much better in terms of user experience! They should keep working across Minecraft versions as well - integers won't.

Do you want to drop support the integer block ids in the longer term? Or are you expecting they will keep working (although we'd prefer Minecraft string block ids)? I'm always conscious of backwards compat and don't like breaking things once they get published. Parsing the two formats shouldn't be a significant issue (only more work on your part). But if the int block ids are going to go away, its worth noting that in doco now.

Reply:
The thing with the integers how it is right now will be removed then.

Reply:
I've added a note warning people about the integers.

Reply:
I've added a note warning people the integers will disappear one day.

Reply:
I've added a note warning people the integers will disappear one day.

Reply:
I've added a note warning people about the integers.